app/videobridge/server/methods/jitsiSetTimeout.js

```diff
@@ -9,7 +9,7 @@ import { canSendMessage } from '../../../authorization/server';
 import { SystemLogger } from '../../../logger/server';
 
 Meteor.methods({
-	'jitsi:updateTimeout': (rid) => {
+	'jitsi:updateTimeout': (rid, joiningNow) => {
```

Comment:
maybe you need to change to have `joiningNow` default value as `true`, otherwise it will break other clients that don't know this new param (like the mobile apps)